packages/plugins/integration-make-com/src/lib/make-com.controller.ts

```
@@ -3,16 +1,23 @@
+import { Controller, Get, Post, Body, UseGuards, BadRequestException, NotFoundException, Logger } from '@nestjs/common';
+import { ConfigService } from '@gauzy/config';
 import { ApiTags, ApiOperation, ApiResponse } from '@nestjs/swagger';
 import { TenantPermissionGuard, Permissions, PermissionGuard, RequestContext } from '@gauzy/core';
 import { PermissionsEnum } from '@gauzy/contracts';
 import { MakeComService } from './make-com.service';
-import { IMakeComIntegrationSettings } from './interfaces/make-com.model';
-import { UpdateMakeComOAuthSettingsDTO, UpdateMakeComSettingsDTO } from './dto';
+import { IMakeComCreateIntegration, IMakeComIntegrationSettings } from './interfaces/make-com.model';
+import { UpdateMakeComSettingsDTO } from './dto';
+import { MakeComOAuthService } from './make-com-oauth.service';
+import * as crypto from 'crypto';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)